generators/typescript/utils/core-utilities/fetcher/src/form-data-utils/FormDataWrapper.ts

```diff
@@ -19,7 +19,7 @@ export interface CrossPlatformFormData {
 
     appendFile(key: string, value: unknown, fileName?: string): Promise<void>;
 
-    getRequest(): MaybePromise<FormDataRequest<unknown>>;
```

Comment:
We usually use `unknown` instead of `any`. Any reason why this was changed here?

Reply:
yes, now fetch type is more strict and no longer accepts a RequestInit<unknown> so many type checks in tests for example fail

That unknown has no reason to exist because there is no way to override it with a generic